ntplib.py

```diff
@@ -280,20 +280,28 @@ def __init__(self):
         """Constructor."""
         pass
 
-    def request(self, host, version=2, port="ntp", timeout=5):
+    def request(self, host, version=2, port="ntp", timeout=5, addressfamily=""):
```

Comment:
How about:
```
address_family=socket.AF_UNSPEC
[...]
addrinfo = socket.getaddrinfo(host, port, family=address_family)[0]
```
This way, no need for `if` statements, and the API uses properly typed variables instead of strings.

Reply:
I think i tried this first, but in my script calling ntplib I would have to `import socket` to make this work. I went the way of a string instead to bypass having to do the import where i would otherwise not use it. 

Either way is good for me, whichever you feel is best

Reply:
I would definitely prefer using `socket.AF_UNSPEC`, thanks :).

Reply:
Hey,

I'll close this MR soon without a patch, in case you're still motivated.